### PR TITLE
Updates for Upbound Cloud GA and UXP

### PR DESCRIPTION
**Pull request by turkenh**
### Description of your changes

This PRs updates the examples and readme for the latest changes in the Upbound Cloud GA and UXP.

I have:

- [x] Read and followed Upbound's [contribution process](https://git.io/fj2m9).

### How has this code been tested

1. Create a hosted cp on Upbound Cloud
2. Follow the updated instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,16 +211,16 @@ docker login ${REGISTRY} -u ${UPBOUND_ACCOUNT_EMAIL}
...
 ```console
-kubectl crossplane build configuration --name package.xpkg --ignore "examples/*,hack/*"
+up xpkg build --name package.xpkg --ignore "examples/*,hack/*"
 ```
 
 Push package to registry.
 
 ```console
-kubectl crossplane push configuration ${PLATFORM_CONFIG} -f package.xpkg
+up xpkg push configuration ${PLATFORM_CONFIG} -f package.xpkg
```
Comment (hasheddan) [inline README.md:220]:
```suggestion
up xpkg push ${PLATFORM_CONFIG} -f package.xpkg
```

File: README.md [hunk 2]
```diff
@@ -34,7 +85,8 @@ cp kubectl-crossplane /usr/local/bin
 #### Install the Platform Configuration
 
 ```console
-PLATFORM_CONFIG=registry.upbound.io/upbound/platform-ref-gcp:v0.0.4
+PLATFORM_VERSION=v0.0.4
```
Comment (stevendborrelli) [inline README.md:88]: Can we update the version here?

Comment (turkenh) [reply README.md:88]: This is the latest available version in this repo: https://github.com/upbound/platform-ref-gcp/tags
Could you elaborate a bit what do you mean by updating?

Comment (stevendborrelli) [reply README.md:88]: Since we are changing all the namespace secret references to `Upbound-system`, we should increment the version.

Comment (turkenh) [reply README.md]: I see, thanks!

**Comment by stevendborrelli** [comment]
LGTM. It will need one of the required reviewers to approve.

**Review by hasheddan — APPROVED** [review]

**Pull request merged**